cpp/harvest/nacjd.cc

```diff
@@ -461,7 +486,9 @@ void getStatistics(int argc, char **argv) {
 
     const auto root_node(JSON::JSONNode::CastToObjectNodeOrDie("tree_root", internal_tree_root));
     unsigned total(0), no_title(0), no_description(0), no_ID(0), no_alternate_ID(0), no_creators(0), no_license(0), no_initial_date(0),
-        no_keywords(0);
+        no_keywords(0), no_statistics_category(0);
+
+    int statistics_count[] = { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
```

Comment:
Instead of using a int[] with a hardcoded length, you should use a more dynamic datatype instead, e.g. a `std::map<std::string, unsigned>` (where the std::string is the name of the category)